data_model/1.5/clusters/WakeOnLAN.xml

```diff
@@ -81,6 +81,9 @@ Davis, CA 95616, USA
       <constraint>
         <desc/>
       </constraint>
+      <constraint>
+        <allowed value="16"/>
+      </constraint>
```

Comment:
Looks like a `desc` constraint in table is being overridden/added because of `ipv6addr` type demanding 16 bytes?

@cecille @hasty

Reply:
will track in an alchemy issue. Hasty is normally pretty responsive to these small fixes. https://github.com/project-chip/connectedhomeip/pull/40846#discussion_r2325254953